runtime/compiler/control/JitDump.cpp

```diff
@@ -551,16 +628,14 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
                   continue;
 
                bool isAOTBody = false;
-               J9JITExceptionTable *metadata = NULL;
-               void *startPC = jittedMethodsOnStack[i]._oldStartPC;
-               if (startPC)
+
+               J9JITExceptionTable *metadata = jitConfig->jitGetExceptionTableFromPC(crashedThread, reinterpret_cast<UDATA>(jittedMethodsOnStack[i]._oldStartPC));
```

Comment:
Does this still handle `NULL == jittedMethodsOnStack[i]._oldStartPC`?

Reply:
It does. Although the API is not documented if 0 is searched for it will return `NULL`. See implementation here:

https://github.com/eclipse/openj9/blob/48b1439ec6b72c81881904767832ad8697d8cc40/runtime/codert_vm/jswalk.c#L1477-L1528

This code isn't performance sensitive so just simplified it to perform the lookup without checking for 0 input.

Reply:
I think either that documentation should be updated to explicitly allow this usage pattern or the test of `startPC` should be restored.

Reply:
Documentation added in 83106bc.